### PR TITLE
Add Editors and a tool

### PR DESCRIPTION
**Pull request by Andersson007**
Add Editors and a tool

Related discussions:
- https://github.com/ansible/ansible-documentation/pull/2284
- https://forum.ansible.com/t/should-we-revamp-the-tools-and-programs-page/39024

### CODE REVIEW

**Comment by Andersson007** [comment]
cc @gundalow

**Review by gundalow — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -96,6 +96,7 @@ For more information about communication, see the [Ansible communication guide](
 - [Ansible Doctor](https://github.com/thegeeklab/ansible-doctor) - Simple annotation like documentation generator for Ansible roles based on Jinja2 templates.
 - [Ansible cmdb](https://github.com/fboender/ansible-cmdb) - Takes the output of Ansible's fact gathering and converts it into a static HTML page.
 - [ARA](https://github.com/ansible-community/ara) - ARA Records Ansible playbooks and makes them easier to understand and troubleshoot with a reporting API, UI and CLI.
+- [Ansible Inventory Grapher](https://github.com/willthames/ansible-inventory-grapher) - Visually displays inventory inheritance hierarchies and at what level a variable is defined in inventory.
```
Comment (oraNod) [inline README.md:99]:
```suggestion
- [Ansible Inventory Grapher](https://github.com/willthames/ansible-inventory-grapher) - Displays inventory inheritance hierarchies and the level at which variables are defined in an inventory.
```

Comment (oraNod) [reply README.md:99]: "Visually displays" is redundant phrasing.

File: README.md [hunk 2]
```diff
@@ -156,5 +157,18 @@ For more information about communication, see the [Ansible communication guide](
...
+  - [lsp-mode](https://emacs-lsp.github.io/lsp-mode/page/lsp-ansible/) - Ansible Language Server Protocol support for Emacs: provides syntax highlighting, auto-completion and diagnostics.
+  - [yaml-mode](https://github.com/yoshiki/yaml-mode) - YAML highlighting and syntax checking.
+  - [jinja2-mode](https://github.com/paradoxxxzero/jinja2-mode) - Jinja2 highlighting and syntax checking.
+  - [magit-mode](https://github.com/magit/magit) - Git porcelain within Emacs.
+  - [flymake-ansible-lint](https://github.com/jamescherti/flymake-ansible-lint.el) - Ansible Lint integration with automatic/continuous annotation of errors, warnings, and info while editing.
+- [PyCharm](https://www.jetbrains.com/pycharm/) - A full IDE for Python software development. Useful plugins include:
+  - [Ansible Lint](https://plugins.jetbrains.com/plugin/20905-ansible-lint) - Ansible Lint integration with automatic/continuous annotation of errors, warnings, and info while editing.
+  - [Ansible Vault Integration](https://plugins.jetbrains.com/plugin/14353-ansible-vault-integration) - Integrate the ansible vault directly into IntelliJ IDEA with context actions for vaulting and unvaulting secrets.
```
Comment (oraNod) [inline README.md:173]:
```suggestion
  - [Ansible Vault Integration](https://plugins.jetbrains.com/plugin/14353-ansible-vault-integration) - Ansible vault integration for IntelliJ IDEA with context actions for vaulting and unvaulting secrets.
```

**Comment by Andersson007** [comment]
@gundalow @oraNod thanks for reviewing!

**Pull request merged**